STM32F4/variants/generic_f407v/ld/common.inc

```diff
@@ -164,11 +150,43 @@ SECTIONS
          * I'm shoving these here naively; there's probably a cleaner way
          * to go about this. [mbolivar]
          */
-        _lm_heap_start = DEFINED(_lm_heap_start) ? _lm_heap_start : _end;
+        _lm_heap_start = DEFINED(_lm_heap_start) ? _lm_heap_start : __ccm_end__;
         _lm_heap_end   = DEFINED(_lm_heap_end) ? _lm_heap_end : __msp_init;
```

Comment:
__msp_init is a value calculated at the top of the linker script, in this line:
https://github.com/victorpv/Arduino_STM32/blob/c4233423541f79e7683d2aa43ebebc2cbc356ca1/STM32F4/variants/generic_f407v/ld/common.inc#L43

What line 153 does is tell the linker that the top of the heap is the start of the stack, which is ORIGIN(ccmram) + LENGTH(ccmram))

__ccm_end__ is misleading since it's not the end of the physical CCM space, but rather the first free address after variables allocated in CCM. I can change it to __ccm_var_end__ unless you think of a more meaningful one.